scripts/lib/libvirt/compute-config

```diff
@@ -32,6 +32,8 @@ def main():
                         help="Virtual Disk Directory (e.g /dev/cloud)")
     parser.add_argument("bootorder", type=str,
                         help="Boot Order (e.g. 2)")
+    parser.add_argument("numcontrollers", type=str,
+                        help="Number of controller nodes")
```

Comment:
@aplanas I didn't make a point. I just told you why strings have been used in the beginning. if argparse casts those somehow, then a type int makes sense, no arguing about that.